docs/adr/0004-translations.md

```diff
@@ -0,0 +1,53 @@
+
+   * **Link and button labels** are not extracted. We think link and button labels are unlikely to change, and extracting them made the application tests harder to read by obfuscating some of the crucial steps in the test with translation keys.
+   * **Big guidance** is extracted into it's own Markdown files and stored alongside the corresponding HTML page that shows it. For example, the guidance for creating a new document is stored in `app/views/new_document/guidance.md`.
+   * **Domain data** that's static is extracted into custom YAML files. This application has two static models (DocumentTypeSchema and SupertypeSchema) that encapsulate domain concepts where the data is part of the application. We think domain data - whether it's a backend setting or a label - should be defined in one place so it can change together. In this application, we've chosen to store the backing files for the two static models in `app/formats`.
+   * **Global strings** (states and validation messages) are extracted using translations. As these strings aren't page-specific, we put them at the top-level of the translation hierarchy (in `states.yml` and `validations.yml`).
+   * **All other strings** are extracted using translations, in a hierarchy that follows the structure of the `app/views` directory. For example, the above example relates to `app/views/publish_document/published.html.erb`.
+
+Every instance of a string in the tests has been replaced according to the above rules, such that the tests continue to pass when an extracted string is changed. **Link and button labels** are not replaced, as they are not extracted in the code.
+
+We also configured Rails to raise an exception when we hit a page where a translation is missing, as we don't test all of the translations; this is done by setting `config.action_view.raise_on_missing_translations = true` in `application.rb`.
+
+## Status
+
+Pending
+
+The approach taken so far is primarily driven by the needs of developers, rather than the needs of the other roles in the team. We should avoid committing to this approach until other people in the team have made more use of it.
+
+## Consequences
+
+Using translations for the most of the strings has allowed members of the team who aren't developers to make changes without a developer or a development environment. Instead, we trained them to use the Github commit and PR interface.
+
+Some people found it hard to find extracted strings, due to them being in multiple, technology-specific locations. It's still unclear how we should structure the extracted strings in a way that makes it easy to find a given string. One way to ignore this problem is to use search (e.g. Github search) to locate the file where the string is stored; this doesn't always work because some strings are only shown under certain conditions (e.g. errors) and therefore may not be widely known.
```

Comment:
Sounds good - I think this is one of our trickier problems. I'm intrigued to learn if content designers get baffled by the controller/action layout but also short of better ideas. 

Only thing I've got is that content-designers might find it easier dealing with a long file with lots in rather than a short one so could jump from controller/action to just a controller.yml - but not yet a known problem or whether that would help.